hail/src/test/scala/is/hail/expr/ir/AggregatorsSuite.scala

```diff
@@ -713,18 +713,6 @@ class AggregatorsSuite extends HailSuite {
     )
   }
 
-  @Test def downsample() {
```

Comment:
Can this test not work under new aggs? Has it been ported to python?

Reply:
A better version of this test exists in Aggregators2Suite. Rather than trying to push abstractions through the old machinery that would let me add support for set comparisons here (order from downsample is not guaranteed), I decided to delete it.